README.md

```diff
@@ -266,11 +266,18 @@ and add a template for source download URL.
     "type": "anitya",
     "project-id": 6377,
     "stable-only": false,
+    "versions": {"<": "1.12.0"},
     "url-template": "https://github.com/flatpak/flatpak/releases/download/$version/flatpak-$version.tar.xz"
 }
 ```
 Set `stable-only` to `true` to retrieve latest stable version (as recognized by Anitya).
 
+You can also set version constraints in `versions` property (optional). It should
```

Comment:
Ideally, we probably should put this paragraph somewhere once, and reference it from descriptions of all checkers that support the constraints, instead of repeating the same text in multiple places.

Reply:
Tried to unify the wording. Is the new version acceptable?